src/Patches/HudManagerPatch.cs

```diff
@@ -44,19 +46,20 @@ public static void CreateHudPercentages()
         [HarmonyPatch(typeof(HUDManager), "Update")]
         public static void Update()
         {
-            if (GameNetworkManager.Instance.localPlayerController == null || _instantiating || _hudPercentagesText == null)
+            PlayerControllerB player_controller = GameNetworkManager.Instance.localPlayerController;
+            if (player_controller == null 
+                || _instantiating 
+                || _hudPercentagesText == null)
             {
                 return;
             }
-
-            float health = Mathf.RoundToInt(GameNetworkManager.Instance.localPlayerController.health);
-            float sprint = Mathf.RoundToInt(((GameNetworkManager.Instance.localPlayerController.sprintMeter * 100f) - 10f) / 90f * 100f);
-            if (sprint < 0f)
-            {
-                sprint = 0f;
-            }
-
-            _hudPercentagesText.text = $"{health}\n\n\n\n{sprint}%";
+            float health = Mathf.RoundToInt(player_controller.health);
+            int sprint = Math.Max(
+                Mathf.RoundToInt(((player_controller.sprintMeter * 100f) - 10f) / 90f * 100f), 
+                0
+                );
```

Comment:
Personal choice: This should either be on one line or have the parameter names prepended to the arguments:

```cs
            int sprint = Math.Max
            (
                val1: Mathf.RoundToInt(((player_controller.sprintMeter * 100f) - 10f) / 90f * 100f),
                val2: 0
            );
```